src/DTO/DefaultProxyFeature.php

```diff
@@ -28,7 +28,7 @@
      *             value: string
      *         }
      *     },
-     *     impression_data: bool
+     *     impressionData: bool
```

Comment:
This should IMO accept both, optionally.

Reply:
Yes